pkg/users/delete-group-dialog.js

```diff
@@ -49,12 +49,16 @@ export function delete_group_dialog(group) {
                 caption: group.userlistPrimary.length > 0 ? _("Force delete") : _("Delete"),
                 style: "danger",
                 clicked: () => {
-                    const prog = ["/usr/sbin/groupdel"];
+                    const prog = ["groupdel"];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230215-180016-4889e9dc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)